hw/constraints_amdc_revd.xdc

```diff
@@ -9,319 +9,324 @@
 # RGB LED #
 ###########
 
-set_property IOSTANDARD LVCMOS18 [get_ports user_led_din];
+set_property IOSTANDARD LVCMOS18 [get_ports {user_led_din[0]}]
 
-set_property PACKAGE_PIN R8 [get_ports user_led_din]; # JX2-14
+set_property PACKAGE_PIN R8 [get_ports {user_led_din[0]}]
```

Comment:
Interesting to see this file updated; was this automatically created via Vivado? Are there any functional changes? If not, let's revert this file since all the comments about net connections on the PicoZed / PCB have been removed in this PR. Unless for good reason, let's keep these comments in this file.